src/World.cpp

```diff
@@ -3051,7 +3051,11 @@ void cWorld::QueueTask(cTaskPtr a_Task)
 }
 
 
-
+void cWorld::ScheduleTask(int a_DelayTicks, std::function<void (cWorld&)> a_Func)
+{
+	cTaskLambda task = cTaskLambda(a_Func);
```

Comment:
This is causing the error with clang, because it indirectly invokes the implicitly auto-generated cTask copy constructor, which is depreciated because of the destructor. You can fix that by either eliminating the copy, by doing `cTaskLambda task(a_Func);` or adding `cTask (const cTask & other) = default;` to cTask.